saleor/webhook/transport/utils.py

```diff
@@ -523,6 +551,93 @@ def clear_successful_delivery(delivery: "EventDelivery"):
         delete_files_from_private_storage_task(files_to_delete)
 
 
+@allow_writer()
+def clear_successful_deliveries(deliveries: list["EventDelivery"]):
```

Comment:
We alreayd have a `clear_successful_delivery`. Now we have separate log for single delviery and separate logic for batch of deliveries. The "core" logic should always be the same. Instead of adding new helper function, I would suggest to modify the existing one to support single and list. Like


```python
def clear_successful_deliveries(delvieries: list []):
    # main logic

def clear_successful_delivery(delivery: "EventDelivery"):
    clear_successful_deliveries([delivery])
```

Same questions to `process_failed_deliveries` and `create_attempts_for_deliveries`. It is not new functionality, it is something that we extend from single object to multiple object. In that case, why we add fully new implementation, instead of making the common part for single and multiple?